build.rs

```diff
@@ -1,17 +1,30 @@
 extern crate gcc;
 
 fn main() {
-    let sources = ::std::fs::read_dir("llvm/lib/Fuzzer")
-        .expect("listable source directory")
-        .map(|de| de.expect("file in directory").path())
-        .filter(|p| p.extension().map(|ext| ext == "cpp") == Some(true))
-        .collect::<Vec<_>>();
-    let mut config = gcc::Config::new();
-    for source in sources.iter() {
-        config.file(source.to_str().unwrap());
+    if let Ok(custom) = ::std::env::var("CUSTOM_LIBFUZZER_PATH") {
+        let custom_lib_dir = ::std::path::PathBuf::from(&custom);
+        let custom_lib_dir = custom_lib_dir.parent().unwrap().to_string_lossy();
+
+        let custom_lib_name = ::std::path::PathBuf::from(&custom);
+        let custom_lib_name = custom_lib_name.file_stem().unwrap().to_string_lossy();
+        let custom_lib_name = &custom_lib_name[3..];
```

Comment:
You cannot assume filename begins with `lib`. Use `str::trim_left_matches` instead.